content/200-orm/500-reference/325-prisma-config-reference.mdx

```diff
@@ -122,46 +122,29 @@ export default {
 ```
 
 ### `schema`
-- Type: `object`
-- Required: No
-- Default: `{ kind: 'single', filePath: './prisma/schema.prisma' }`
-
-Configures how Prisma ORM locates and loads your schema file(s). See sub-options below for details.
-
-#### `schema.filePath`
-- Type: `string`
-- Required: Only when `schema.kind` is `'single'`
-- Default: `'./prisma/schema.prisma'`
-
-Specifies the path to your Prisma schema file when using a single-file setup.
 
-#### `schema.folderPath`
 - Type: `string`
-- Required: Only when `schema.kind` is `'multi'`
+- Required: No
 - Default: none
 
-Specifies the path to the folder containing your Prisma schema files when using a multi-file setup.
-
-#### `schema.kind`
-- Type: `'single' | 'multi'`
-- Required: Only if `schema` is specified
-- Default: `'single'`
+Configures how Prisma ORM locates and loads your schema file(s). See sub-options below for details.
```

Comment:
```suggestion
Configures how Prisma ORM locates and loads your schema file(s). Can be a file or folder path. Relativ paths are resolved relative to the `prisma.config.ts` file location. See [link to schema location docu] for more info about schema location options.
```

Reply:
Might also be worth to have a comment here that `--schema` takes precedence over this option?